.gitmodules

```diff
@@ -1,7 +1,3 @@
-[submodule "jsk_apc2016_common/python/jsk_apc2016_common/rbo_segmentation"]
-	path = jsk_apc2016_common/python/jsk_apc2016_common/rbo_segmentation
-	url = https://github.com/yuyu2172/rbo-apc-object-segmentation
- 	branch = jsk_apc
```

Comment:
This says we uses `jsk_apc` branch.
However, I think you copied `apc2016` branch.
Which is correct?

Reply:
I understand #1776 forgot to change `.gitmodules` and `.gitmodules` is wrong.